repair/repair.cc

```diff
@@ -419,13 +422,17 @@ size_t tracker::nr_running_repair_jobs() {
     return count;
 }
 
+bool tracker::is_aborted(const utils::UUID& uuid) {
+    return _aborted_pending_repairs.contains(uuid);
```

Comment:
Here too, instead of this _aborted_pending_repairs, could we just have a status ABORTED_PENDING? (or maybe just ABORTED? what happens if you abort a repair that wasn't pending - wouldn't it be good to know it was aborted and not "failed"?)

Reply:
By the way, when a repair is aborted, where does its _status change (and to what)? don't we need to change it in case the user wants to ask what happened to a specific repair id?

Reply:
When a repair job is aborted. It will be in FAILED status. 

The 3 status: RUNNING, SUCCESSFUL, FAILED are part of the API.  It is better not to change the API to surprise users, especially for a "fix" patch.